hello_world.py

```diff
@@ -1 +1,34 @@
-print("hello world")
+# importing random to make use of it for tuples
+import random
+
+print("Hello World")
+
+# Closes Issue: #9
+# Asks the user to enter their name and displays it to them
+userName = input("Enter your name: ")
+print("Hello %s!" % userName)
+
+# Closes Issue: #8
+# Picks a random fruit from the tuple and tells the user to eat it
+fruits = ("apple", "banana", "cherry", "strawberry", "pineapple", "mango",
+          "grape", "orange")
+print("You should eat a %s." % fruits[random.randint(0, len(fruits))])
+
+# Closes Issue: #6
+# Creates an array of car names
+cars = ["Ford", "Toyota", "Honda", "Mazda", "Tesla", "BMW"]
+
+# Closes Issue: #4
+# Will make use of arithmetic operations by trying to guess the user's age
+print("The computer will guess your age.")
+age = int(input("Pick a number between 1 and 10: "))
```

Comment:
Whoa, this was mind blowing! 👍 

Do you happen to have any explanation on how this works?

Reply:
@shawnkoon I watched a YouTube video on how people guess someone's age so I can understand the algorithm for it.